### PR TITLE
File size check before upload

### PR DESCRIPTION
**Pull request by komalahuwalia**
Before your pull request is reviewed and merged, please ensure that:

* [ ] there are no linting errors
* [ ] your code is in a uniquely-named feature branch and has been rebased on top of the latest master. If you're asked to make more changes, make sure you rebase onto master then too!
* [ ] your pull request is descriptively named and links to an issue number, i.e. `Fixes #123`
* [ ] **Make sure you have run the tests!!**

Thank you!

### CODE REVIEW

File: server/integrationTest/userFolderSizeJourneyTest.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    const loginResponse = await request
+      .post('/api/auth/login')
+      .set('Accept', 'application/json')
+      .send(loginCreds);
+
+    expect(loginResponse.header['set-cookie'][0]).contains('peblioSession-test');
+    expect(loginResponse.statusCode).to.equal(200);
+
+    const getFileInfoResponse = await request
+      .get('/api/files/size')
```
[the remark this answers is not in the document]
Comment (komalahuwalia) [reply server/integrationTest/userFolderSizeJourneyTest.js]: The filesize is being tested for pebliotest user. We use the same user for authentication test flow

**Comment by MathuraMG** [comment]
@komalahuwalia - can we fix the merge conflicts and merge to production

**Comment by MathuraMG** [comment]
@komalahuwalia - can we fix the merge conflicts and merge to production

**Comment by MathuraMG** [comment]
Let's go ahead and merge this

**Pull request merged**